jnigi.go

```diff
@@ -1082,6 +1082,7 @@ func (o *ObjectRef) CallMethod(env *Env, methodName string, returnType interface
 		retVal = float64(callDoubleMethodA(env.jniEnv, o.jobject, mid, jniArgs))
 	case rType == Object || rType.isArray():
 		obj := callObjectMethodA(env.jniEnv, o.jobject, mid, jniArgs)
+		refs = append(refs, obj)
```

Comment:
Looks good but only should happen if the conversion to Go slice is going to happen.  This statement should be moved to line1090 after `arrayToConvert = obj`. Hope that will still pass test.

Reply:
Fixed